src/components/Business/Business.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import './Business.css';
 
+
+
```

Comment:
Remove extra spaces like this from the file

Reply:
@rajat2502 done